org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/xsi/XSISchemaModel.java

```diff
@@ -246,6 +247,9 @@ else if(name.endsWith(":type")) {
 			}
 		}
 
-		return doc;
+		MarkupContent content = new MarkupContent();
+		content.setKind(MarkupKind.MARKDOWN);
```

Comment:
Is there a reason why the kind is not coming from the hover request?

Reply:
To be honest with you, I don't know this code. It seems all documentation are written in Markdown. Please create an issue for that. Thanks.

Reply:
Created: https://github.com/eclipse/lemminx/issues/732